dronecan/sensors/rc/1141.RCLinkStats.uavcan

```diff
@@ -0,0 +1,38 @@
+#
+# This definition describes metadata and statistics for a RF control link.
+# It provides information regarding a link's signal quality, as well as transmitter
+# power, and antenna information. It accomodates single-radio, and dual-radio
+# link architectures. It provides information for both uplink (control unit to airborne
+# receiver), and downlink (airborne receiver to control unit) data.
+#
+
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+
+# Uplink - received signal strength (RSSI) for antennas 1 and 2 respectively. Ranges from 
+# -255 to 0. (An integer value of 100 means RSSI is -100dBm)
+uint8 uplink_rssi_1
+uint8 uplink_rssi_2
+
+# Link quality, on a scale from 0 to 255 - its precise definition may change basd on the link 
+# protocol used. Example: percentage of recent packets received correctly. With this example, 
+# 0 means no valid packets received recently; 255 means 100% of packets received recently are # valid.
+uint8 uplink_link_quality
```

Comment:
FWIW it is better to provide something that can be used by a recipient without having to know the details, or providing the details. I usually go with the former case. Here you might say its a linearly mapped percentage - so 1 is all packets lost, 100% is all packets received/perfect, 50% is half of packets lost. 

Something like:

```suggestion
# Link quality, on a scale from 1 to 255, mapped linearly as a percentage.
# (the actual method used to evaluate quality depend on the protocol,
# but is generally a measure of packet loss).
# 0 means data cannot be provided; 1 to 255 are mapped to  1 to 100%.
uint8 uplink_link_quality
```